frontend/src/scenes/trends/trendsLogic.ts

```diff
@@ -304,7 +304,7 @@ export const trendsLogic = kea<
     selectors: () => ({
         filtersLoading: [
             () => [eventDefinitionsLogic.selectors.loaded, propertyDefinitionsLogic.selectors.loaded],
-            (eventsLoaded, propertiesLoaded) => !eventsLoaded || !propertiesLoaded,
+            (eventsLoaded, propertiesLoaded): boolean => !eventsLoaded || !propertiesLoaded,
```

Comment:
nit: it should be okay without the annotation, right?

Reply:
Not entirely sure, without the annotation this selector was identified as `any` in the UI component, instead of `boolean`. Might've been my typegen not refreshing 🤷